index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="style.css" />
+  <title>Cookie Stand</title>
+</head>
+<header></header>
+<nav></nav>
+<body>
+  <h1 id="highlight">Salmon Cookie Stand</h1>
```

Comment:
All of these html elements are squished together quite a bit, and it's pretty tough to read. Some spacing based on each store would make this a lot more readable later.